### PR TITLE
Feature: Moved "format drive" to main context menu

### LINKED ISSUE #11284: Feature: Option to format drives should be displayed in the main menu
### What feature or improvement do you think would benefit Files?

The option to format drives is currently in the overflow menu, we should make it more accessible and move it to the main menu.

### Requirements

- Display option to format drives in the main menu when right clicking them
- This should apply to the sidebar and drives widget

### Files Version

v2.4.34

### Windows Version

Windows 11

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11284

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -2955,4 +2955,7 @@
   <data name="ShowHiddenItems" xml:space="preserve">
     <value>Show hidden items</value>
   </data>
+  <data name="FormatDriveText" xml:space="preserve">
+    <value>Format...</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:2959]: Why the "..."?

Comment (hecksmosis) [reply src/Files.App/Strings/en-US/Resources.resw:2959]: That's the default string used by windows (I suppose it's there because it pops up a menu)

**Comment by yaira2** [comment]
Can you do this with BaseLayout as well?

**Comment by hecksmosis** [comment]
> Can you do this with BaseLayout as well?

When would this be used?

**Comment by yaira2** [comment]
When right clicking a blank space

**Comment by hecksmosis** [comment]
Ok

**Comment by hecksmosis** [comment]
Done

**Review by yaira2 — APPROVED** [review]
LGTM

**Comment by yaira2** [comment]
Does right clicking a folder inside a drive crash the app for you?

**Comment by yaira2** [comment]
The crash is fixed but the menu doesn't load when right clicking a selected folder.

**Comment by hecksmosis** [comment]
Yes, I know, this is because of an bug in a merge in #11365 with #11357

**Comment by hecksmosis** [comment]
Fixed

**Review by yaira2 — APPROVED** [review]
LGTM

**Comment by yaira2** [comment]
> When right clicking a blank space

I just checked and this is actually not a valid use case.

**Pull request merged**